metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/sql/DirectSqlGetPartition.java

```diff
@@ -74,6 +76,12 @@
 public class DirectSqlGetPartition {
     private static final String FIELD_DATE_CREATED = "dateCreated";
     private static final String FIELD_BATCHID = "batchid";
+    private static final String WAP_TABLE_DB = "audit";
+    private static final Pattern WAP_TABLENAME_PATTERN = Pattern.compile(
+        "(?<db>.*)__(?<table>.*)__audit(.*)$"
```

Comment:
This can be "(?<db>.*)__(?<table>.*)__.*"?. Also the variable name can be AUDIT_TABLENAME_PATTERN.

Reply:
Agreed.